devicetypes/smartthings/zwave-basic-window-shade.src/zwave-basic-window-shade.groovy

```diff
@@ -147,13 +163,21 @@ def installed() {
 def updated() {
 	sendHubCommand(pause())
 	state.reverseDirection = reverseDirection ? reverseDirection : false
+	int time = calibrationTime
+
+	if (calibrationTime >= 5 && calibrationTime <= 255) {
```

Comment:
did you mean to be checking `time` instead of `calibrationTime` here? As far as I can tell they should be equal, though, so I don't know if having a second variable is worthwhile.

Reply:
I agree that we should just use `calibrationTime` here.

Reply:
@greens @cbaumler - the "int time = calibrationTime" was primarily with me messing around and practicing a little bit while figuring out how preferences are used as numbers/integrrs, i ended up leaving it in there and forgot to remove it when i realized calibrationTime was pretty much a local variable for the device handler.

I'll have that removed.

Reply:
@greens @cbaumler - edited for this function to only use calibrationTime number.